### PR TITLE
Exposed editor brush texts to localisation

### PR DESCRIPTION
**Pull request by Mailaender**
Noticed during https://github.com/OpenRA/OpenRA/pull/20842.

### CODE REVIEW

File: mods/common/languages/en.ftl [hunk 1]
```diff
@@ -717,3 +717,24 @@ options-time-limit =
...
+       [one] Added { $amount } cell of { $type }
+      *[other] Added { $amount } cells of { $type }
+    }
+## EditorTileBrush
+notification-added-tile = Added tile { $id }
+notification-filled-tile = Filled with tile { $id }
+## EditorActionManager
+notification-opened = Opened
+## ActorEditLogic
+notification-edited-actor = Edited { $name } ({ $id })
```
Comment (PunkPun) [inline mods/common/languages/en.ftl:750]: the current file style is to have whitespaces before comments

Comment (Mailaender) [reply mods/common/languages/en.ftl:750]: They are all the same (editor brushes).

Comment (PunkPun) [reply mods/common/languages/en.ftl:750]: That's the current style though. If we are to change style we should apply it to all en.ftl files

**Review by PunkPun — COMMENTED** [review]
Otherwise LGTM

File: mods/common/languages/en.ftl [hunk 2]
```diff
@@ -717,3 +717,26 @@ options-time-limit =
...
+## EditorDefaultBrush
+notification-removed-actor = Removed { $name } ({ $id })
+notification-removed-resource = Removed { $type }
+## EditorResourceBrush
+notification-added-resource =
+    { $amount ->
+       [one] Added { $amount } cell of { $type }
+      *[other] Added { $amount } cells of { $type }
+    }
+## EditorTileBrush
```
Comment (PunkPun) [inline mods/common/languages/en.ftl:742]: Whitespace wasn't added to all entries

File: mods/common/languages/en.ftl (hunk 1 above)
Comment (Mailaender) [reply mods/common/languages/en.ftl:750]: Adding the white spaces is not too bad.

File: OpenRA.Mods.Common/EditorBrushes/EditorCopyPasteBrush.cs [hunk 3]
```diff
@@ -205,7 +208,7 @@ class CopyPasteEditorAction : IEditorAction
 			mapHeight = map.Height;
 			mapResources = map.Resources;
 
-			Text = $"Copied {tiles.Count} tiles";
+			Text = Text = TranslationProvider.GetString(CopiedTiles, Translation.Arguments("amount", tiles.Count));
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/EditorBrushes/EditorCopyPasteBrush.cs]:
```suggestion
			Text = TranslationProvider.GetString(CopiedTiles, Translation.Arguments("amount", tiles.Count));
```

File: OpenRA.Mods.Common/EditorBrushes/EditorDefaultBrush.cs [hunk 4]
```diff
@@ -120,7 +123,8 @@ public RemoveActorAction(EditorActorLayer editorActorLayer, EditorActorPreview a
 			this.editorActorLayer = editorActorLayer;
 			this.actor = actor;
 
-			Text = $"Removed {actor.Info.Name} ({actor.ID})";
+			Text = TranslationProvider.GetString(RemovedActor,
+				Translation.Arguments("name", actor.Info.Name,	"id", actor.ID));
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/EditorBrushes/EditorDefaultBrush.cs]:
```suggestion
				Translation.Arguments("name", actor.Info.Name, "id", actor.ID));
```

File: mods/common/languages/en.ftl [hunk 5]
```diff
@@ -717,3 +717,30 @@ options-time-limit =
         }
 
 notification-time-limit-expired = Time limit has expired.
+
+## EditorActorBrush
+notification-added-actor = Added { $name } ({ $id })
+
+## EditorCopyPasteBrush
+notification-copied-tiles = Copied { $amount } tiles
```
Comment (abcdefg30) [inline mods/common/languages/en.ftl]: This should also get the `1 tile` vs. `X tiles` distinction.

File: mods/common/languages/en.ftl [hunk 6]
```diff
@@ -717,3 +717,30 @@ options-time-limit =
...
+
+## EditorDefaultBrush
+notification-removed-actor = Removed { $name } ({ $id })
+notification-removed-resource = Removed { $type }
+
+## EditorResourceBrush
+notification-added-resource =
+    { $amount ->
+       [one] Added { $amount } cell of { $type }
+      *[other] Added { $amount } cells of { $type }
```
Comment (abcdefg30) [inline mods/common/languages/en.ftl:739]: This doesn't appear to work:
grafik

**Review by abcdefg30 — CHANGES_REQUESTED** [review]

File: mods/common/languages/en.ftl (hunk 6 above)
Comment (PunkPun) [reply mods/common/languages/en.ftl:739]: `one` doesn't seem to be working anymore. You have to use `1`. Did it break in our current fluent version?

Comment (Mailaender) [reply mods/common/languages/en.ftl:739]: Not just here. Created https://github.com/OpenRA/OpenRA/issues/20882.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/885ac4f61c3a881a37c4be14abf87edf9ec3b3c2...5d8428ddb619c6f22ed71dc7d8612054b2874be4)

**Pull request merged**